README.md

```diff
@@ -4,6 +4,11 @@ Rizzo is a heavily customized Vagrant configuration and work flow with a
 role based focus. It is meant to make working with Vagrant easier and
 purpose built for layered Puppet control repositories.
 
+Rizzo loads a personal configuration file, `~/.rizzo.json` by default, which
+lists one or more control repositories.  Rizzo then looks for a loads a
+`.rizzo.json` configuration file located at the root of each listed control
```

Comment:
it only loads from the top layer control repo

Reply:
Right, I keep thinking it merges them for some reason.